CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+- Fix: DEFAULT_ATTRIBUTE_TYPE is changed to Text (#810)
```

Comment:
```suggestion
- Fix: default attribute type changed from 'string' to 'Text' (#810)
```

That wording is "closer" to the user (DEFAULT_ATTRIBUTE_TYPE is for developers).